include/LightGBM/boosting.h

```diff
@@ -80,6 +80,12 @@ class Boosting {
   virtual void ModelsFromString(const std::string& model_str, int num_used_model) = 0;
 
   /*!
+  * \brief Calculate feature importances
+  * \param last_iter Last tree use to calculate
+  */
+  virtual std::string FeatureImportance(int last_iter) const = 0;
```

Comment:
I think this is not need to expose as interface of boosting.h . just using as private member function in gbdt class